### ISSUE TITLE
textract (>=1.6.5,<2.0.0) requires xlrd (>=1.2.0,<1.3.0).

### ISSUE THREAD

**Issue by dickreuter** [deanmalmgren/textract#544]
Because no versions of textract match >1.6.5,<2.0.0
 and textract (1.6.5) depends on xlrd (>=1.2.0,<1.3.0), textract (>=1.6.5,<2.0.0) requires xlrd (>=1.2.0,<1.3.0).
So, because sparkai-chatbot depends on both xlrd (^2.0.1) and textract (^1.6.5), version solving failed.

**Comment by StevenMapes** [comment]
Yeah this ties in with some comments on this PR https://github.com/deanmalmgren/textract/pull/543 which is about the fork Kyle maintains but that version is also still using an old version of xlrd. The suggestion is to migrate the XLSX code to use another library which will then allow the project to hopefully move to a newer version of xlrd but it needs a further PR

**Comment by janatacom2025-ship-it** [comment]
[temp7081263757188467621.PDF](https://github.com/user-attachments/files/23465630/temp7081263757188467621.PDF)
